### DISCUSSION TITLE
opencv-python

### DISCUSSION THREAD

**Discussion by BikonLI** [flet-dev/flet#3200] [Packages] [answered]
Add opencv to flet!

**Comment by ProPython007** [comment]
Sorry to disturb, but when this cv2 release will come in (flet build apk)? Any estimates?

**Comment by BikonLI** [comment]
I have no idea about that. The official blog said it will come in few weeks, but now it seems like it won't be available recently.
I'm already give up on flet, because of it's large size of apk, annoying to debug, trouble to package for android.
Maybe you can try some other framework such as pyqt or kivy before the new package system comes.













At 2024-08-15 04:47:28, "Sayan Karmakar" ***@***.***> wrote:

Sorry to disturb, but when the release will come (flet build apk)? Any estimates?

**Comment by brnhrrldprz** [comment]


**Reply by brnhrrldprz** [reply]
Up

**Comment by FeodorFitsner** [comment, marked as answer]
[`opencv-python` and other packages](https://pypi.flet.dev/) are available in Flet 0.25.0 pre-release and up. [Give it a try](https://flet.dev/blog/flet-new-packaging-pre-release) and let us know how that works.
